### ISSUE TITLE
Unable to run

### ISSUE THREAD

**Issue by ticolensic** [e-hengirmen/Basic-number-captcha-solver#8]
Could you provide tensorflow, keras, protobuf and python versions to run this model? Various versions give random errors :(

Happy new year!

**Comment by e-hengirmen** [comment]
Yeah no problem I have added a requirements.txt also it didnt work with python3.13 so I had to downgrade to 3.10 u can use that version with given requirements.txt
